includes/api/class-wc-rest-product-variations-controller.php

```diff
@@ -131,33 +131,32 @@ public function add_product_id( $args, $request ) {
 	 */
 	public function prepare_item_for_response( $post, $request ) {
 		$variation = wc_get_product( $post );
-		$post_data = get_post( $variation->get_variation_id() );
 		$data      = array(
 			'id'                 => $variation->get_variation_id(),
```

Comment:
`get_variation_id()` got deprecated and just replaced with `get_id()` which will always return the variation ID (get_parent_id will return parent).